instat/dlgOneWayFrequencies.vb

```diff
@@ -81,7 +75,6 @@ Public Class dlgOneWayFrequencies
         ucrPnlFrequencies.AddToLinkedControls(ucrChkFlip, {rdoGraph, rdoBoth}, bNewLinkedDisabledIfParameterMissing:=True, bNewLinkedAddRemoveParameter:=True)
 
         ucrNudGroups.SetParameter(New RParameter("auto.group", 9))
-        ucrNudGroups.SetParameterIncludeArgumentName(True)

```

Comment:
Yes if you set it to False they do not appear. The default is True so if you want it as True you don't have to do anything.